Examples/MagicWeatherSwiftUI/Magic Weather SwiftUI.xcodeproj/project.pbxproj

```diff
@@ -439,6 +436,30 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		4E3BE8642BF77F420045B504 /* XCRemoteSwiftPackageReference "purchases-ios" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/RevenueCat/purchases-ios";
```

Comment:
I think we want to change to use `https://github.com/RevenueCat/purchases-ios-spm` instead, since it's much faster. cc @jamesrb1

Reply:
updated in the latest commit